qhub/schema.py

```diff
@@ -364,6 +364,27 @@ class Main(Base):
     environments: typing.Dict[str, CondaEnvironment]
     clearml: typing.Optional[ClearML]
 
+    @validator("project_name")
+    def project_name_convention(cls, value: typing.Any):
+        convention = """
+        In order to successfully deploy QHub, there are some project naming conventions which need
+        to be followed. First, ensure your name is compatible with the specific one for
+        your chosen Cloud provider. In addition, the QHub project name should also obey the following
+        format requirements:
+        - letters from A to Z (upper and lower case) and numbers;
+        - Special characters are not allowed, such as "!@#$%^&*()";
+        - Maximum accepted length of the name string is 16 characters.
+        - If using AWS: names should not start with the string "aws";
```

Comment:
Where does this come from? It looks like the current aws test has a project name `aws-test`

Reply:
It is the description, but I don't remember why we need to avoid that. I remember a discussion about that a really long time ago... Anyways I will need to add another step in that validation because the [Azure storage name](https://github.com/toddkitta/azure-content/blob/master/articles/guidance/guidance-naming-conventions.md#naming-rules-and-restrictions) must not contain any dashes